lang/python/python3-flent/Makefile

```diff
@@ -0,0 +1,44 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python3-flent
+PKG_VERSION:=2.0.1
```

Comment:
I should really tag a new version, there are a couple of bugs in this one...

Reply:
That would be awesome! :)